dasharo-compatibility/dcu.robot

```diff
@@ -42,7 +42,7 @@ DCU001.001 Change the UUID
     Flash Firmware    ${FW_COPY}
     Make Sure New Firmware Is Booted After Flashing
 
-    Power On
```

Comment:
Do we get rid of `Power On` now? THere are still hundreds of occurrences of Power On that should be changed to `Power Cycle On` then... Why do we change `Power On` to `Power Cycle On` only in selected places?